cmd/apprepository-controller-new/script.txt

```diff
@@ -0,0 +1,99 @@
+These are the commands I ran after bootstrapping kubebuilder locally per
```

Comment:
Great, thanks!

At some point, it'd be great if this file gets converted into a markdown-based one.

Reply:
agreed. at some point 😄 right now I just want to plow ahead, there is a lot of changes to come in